css/index.css

```diff
@@ -1,3 +1,529 @@
+    }
+
+    main .top .features-and-about {
+       width: 760px;
+    }
+
+    main .top .features-and-about p {
+        width: 350px;
+        margin: 0;
+        padding: 0;
+    }
+
+    main .middle {
+        width: 760px;
+        margin-top: 10px;
+        display: flex;
+        flex-direction: column;
+	    justify-content: center;
+        align-items: center;
+    }
+
+    main .middle .middle-img {
+        width: 760px;
+    }
+
+    main .bottom {
+        width: 760px;
+	    display: flex;
+	    justify-content: space-between;
+	    align-items: center;
+	    flex-wrap: wrap;
+        padding-bottom: 36px;
+    }
+
+    main .bottom .services, .product, .vision {
+        margin-top: 30px;
+    }
+
+    main .bottom .services {
+        width: 215px;
+        margin: 30px 0 0 5px;
+        padding: 0;
+    }
+
+    main .bottom .product {
+        width: 215px;
+	    margin: 30px 0 0 0;
+        padding: 0;
+    }
+
+    main .bottom .vision {
+        width: 215px;
+	    margin: 30px 25px 0 0;
+        padding: 0;
+    }
+    
+    .contact {
+        width: 760px;
+        margin-left: 20px;
+	    display: flex;
+	    flex-direction: column;
+	    justify-content: flex-start;
+    }
+
+    .contact h2 {
+	    margin: 39px 10px 15px 10px;
+    }
+
+    .contect address, p {
+        margin-left: 10px;
+    }
+
+    footer {
+        width: 740px;
+        margin: 28px;
+	    display: flex;
+	    justify-content: center;
+    }
+}
+
+/* ================================== phone */
+
+@media (max-width: 500px) {
+        .container {
+        width: 460px;
+        margin: 20px auto;
+        display: flex;
+        flex-direction: column;
+    }
+
+    nav {
+        width: 480px;
+        flex-direction: column-reverse;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    nav .nav-items {
+        width: 480px;
+        margin: 20px 0;
+        display: flex;
+        flex-direction: column;
+        border-bottom: 1px solid darkslategrey;
+    }
+
+    nav .nav-items a {
+        width: 460px;
+        padding: 20px;
+        border-top: 1px solid darkslategrey;
+        font-size: 18px;
+        text-align: center;
+    }
+
+    nav .logo {
+        width: 183px;
+        margin: 10px;
+        align-items: center;
+    }
+
+    header {
+	    width: 460px;
+	    display: flex;
+	    justify-content: center;
+        align-items: center;
+        margin: 20px 0;
+        padding-bottom: 8px;
+        border-bottom: 1px solid black;
+    }
+
+    header .getting-started {
+	    width: 460px;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-around;
+        align-items: center;
+        text-align: center;
+    }
+
+    header .getting-started .start {
+        font-size: 10;
+        margin: 10px auto 30px auto;
+	    padding: 8px 45px;
+	    border: 1px solid black;
+    }
+
+    header .snippet {
+        display: none;
+    }
+
+    main {
```

Comment:
The text contained by child elements of main gets a little cut off on the left in mobile view. Adding a little padding or margin to main will prevent this. :)